mitiq/cdr/cdr.py

```diff
@@ -74,22 +74,29 @@ def execute_with_cdr(
         scale_noise: scale_noise: Function for scaling the noise of a quantum
             circuit.
         scale_factors: Factors by which to scale the noise.
+
             - When 1.0 is the only scale factor, the method is known as CDR.
+
             - Note: When scale factors larger than 1.0 are provided, the method
-            is known as "variable-noise CDR."
+              is known as "variable-noise CDR."
+
         kwargs: Available keyword arguments are:
+
             - method_select (string): Specifies the method used to select the
```

Comment:
```suggestion
            method_select (string): Specifies the method used to select the
```

Reply:
You can also get rid of the manually typed typehints like `(string)` etc. These should show up in a clickable format when the API-doc is built.

Reply:
Is that true? My understanding was that the python typehints inform sphinx, but since items documented in the `kwargs` section do not have any python typehints, where would the information come from?

Reply:
Good point. I completely forgot that the item typehints are not documented. We could try using this: https://peps.python.org/pep-0692/#specification in the future.

Perhaps this is out of scope of this PR.

Reply:
> Perhaps this is out of scope of this PR.

agreed.